src/components/inputs/rich-text-input/editor.js

```diff
@@ -129,6 +134,11 @@ const renderEditor = (props, editor, next) => {
   return (
     <Editor editor={editor} {...passedProps}>
       {children}
+      <HiddenInput
```

Comment:
Is this like a focus trap?

Reply:
Can you explain what a focus trap is?

Reply:
It's a trap for the page's focus or the user's ability to tab around. So only any child within the focus-trap can be focussed. I wasn't sure if this was what we're doing here just couldn't derive fromt the name `HiddenInput`.

Reply:
To understand this, we need to step back.

When you write this in HTML, what does it mean?

```<label htmlFor="name">Name</label>```

It associates this label with the text "Name", with the `<input>` who's id is "name". Right?

<input>. Not div. Not span. You can't associate a label with a <div> or a <span>

**That's the problem.**

Before we were not rendering a <div>, we were rendering an `<output>`.

You **can** associate labels with <outputs> 

However, contenteditable outputs break in Safari.

Reply:
https://jsbin.com/nuqetifoli/edit?html,output

Reply:
Ah got it. Thanks. So it's just the "label-anchor" or something like that.